### PR TITLE
[@types/react] Warn in comment that interfaces in global.d.ts are empty

### PR DESCRIPTION
**Pull request by rauschma**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://twitter.com/rauschma/status/1181852019264503808?s=20

### CODE REVIEW

**Review by eps1lon — COMMENTED** [review]
I'm all in favor of more type documentation.

Not sure how that whitespace lint error got there though. Does it pass local `npm run lint react`?

**Comment by rauschma** [comment]
Not sure how the weird spaces got in there; fixed now. `npm run lint` works now, `npm run react` doesn’t:

```
> dtslint types "react"

Error: /Users/rauschma/tmp/DefinitelyTyped/types/react/index.d.ts:28:22
ERROR: 28:22  expect  TypeScript@next compile error: 
Cannot find module 'csstype'.

    at /Users/rauschma/tmp/DefinitelyTyped/node_modules/dtslint/bin/index.js:190:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/rauschma/tmp/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
```

File: types/react/global.d.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@
 React projects that don't include the DOM library need these interfaces to compile.
 React Native applications use React, but there is no DOM available. The JavaScript runtime
 is ES6/ES2015 only. These definitions allow such projects to compile with only `--lib ES6`.
+
+Warning: all of these interfaces are empty. If you want type definitions for various properties
+(such as HTMLInputElement.prototype.value), you need to add `--lib DOM` (via command line or tsconfig.json).
```
Comment (Jessidhia) [inline types/react/global.d.ts]: The name of the DOM lib is all lowercase: `typescript/lib/lib.dom.d.ts`

**Comment by Jessidhia** [comment]
@rauschma You need to run `npm run test` so the dtslint test suite runs all of the needed sub-package install scripts.

Sometimes that breaks horribly and I find it's easier to `find -type d -name node_modules -exec rm -rf {} \;`  and start over from the `npm install && npm test`.

File: types/react/global.d.ts (hunk 1 above)
Comment (rauschma) [reply types/react/global.d.ts]: This is my source: https://www.typescriptlang.org/docs/handbook/compiler-options.html

**Review by eps1lon — APPROVED** [review]

**Pull request merged**